tests/_cli/snapshots/unsafe-doc.md.txt

```diff
@@ -18,14 +18,48 @@ print("Hello, World!")
 
 ```marimo run convert document.md```
 
+```{.python.marimo unparsable="true"}
+it's an unparsable cell
+```
+
 <!-- Actually markdown -->
 ```{python} `
```

Comment:
is this also syntax i wasn't aware of? like an one-line code-fence?

Reply:
No this is intentionally broken to make sure match works. The newest unit tests turned into more of smoke tests, so yeah- I think more granular tests would be awesome

Reply:
oh gotcha, ha makes more sense